components-api/src/main/scala/pl/touk/nussknacker/engine/api/context/ValidationContext.scala

```diff
@@ -84,8 +82,8 @@ object OutputVar {
   def enricher(outputName: String): OutputVar =
     OutputVar(EnricherFieldName, outputName)
 
-  def subprocess(outputName: String): OutputVar =
-    OutputVar(SubprocessFieldName, outputName)
+  def fragmentOutput(outputName: String, varName: String): OutputVar =
```

Comment:
why it is "ref.outputVariableNames"?  what ref it means?

Reply:
I added comment, it's path to appropriate field in json model